src/pages/study-page/study-page.scss

```diff
@@ -3,9 +3,27 @@
 
 .study-page {
   .study-page-content {
+    width: 500px;
```

Comment:
what's this do? if it's something w the responsiveness breaking at <500px, I believe rebasing on main should fix it

Reply:
It was for styling the icon, but I fixed it so it is bound to the flashcard below.